optimismo/patterns/homepage.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Title: Header
+ * Slug: optimismo/homepage
+ * Categories: featured
+ */
+?>
+
+<!-- wp:group {"align":"full","style":{"dimensions":{"minHeight":"100vh"},"spacing":{"padding":{"top":"0","right":"0","bottom":"0","left":"0"}}},"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"space-between","justifyContent":"stretch"}} -->
+<div class="wp-block-group alignfull" style="min-height:100vh;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0">
+	<!-- wp:spacer {"height":"1px"} -->
+	<div style="height:1px" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer -->
+
+	<!-- wp:group {"align":"full","layout":{"type":"constrained"}} -->
+	<div class="wp-block-group alignfull">
+		<!-- wp:template-part {"slug":"header","theme":"optimismo","tagName":"header","align":"full"} /-->
```

Comment:
This patterns has a reference to a template part which we can't do.  We will need to find another way to accomplish this.

Reply:
I moved this content to the template (instead of a pattern) and that eliminated the issue.